geometry2/f2568f5/f2568f5.bug

```diff
@@ -45,7 +46,7 @@ fix:
   fix-in:
     - tf2_sensor_msgs/package.xml
   languages:
-    - package.xml
+    - Package XML
```

Comment:
Have these new values been noted somewhere?

Reply:
They haven't. I'm updating these values across the dataset for the sake of the analysis at the moment. I'm only including changes to `geometry2` in this PR. Below is a count of the frequency of each value for `fix.languages`:
```
Counter({'C++': 110, 'CMake': 26, 'Python': 23, 'Package XML': 23, 'Xacro/URDF': 18, 'XML': 8, 'Launch XML': 7, 'YAML': 5, 'C': 4, '+': 4, 'python': 4, 'launch XML': 4, 'URScript': 3, 'srv': 1, 'collada': 1, 'shell': 1, 'ASCII STL': 1, 'yaml': 1, 'RViz': 1, 'ROS Message': 1, 'udev rules': 1, 'ROS Param': 1})
```

Reply:
I'll open up a separate issue for this.